plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraSqlExecutor.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.cassandra;
+
+import io.trino.testing.sql.SqlExecutor;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.IntStream;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.joining;
+
+public class CassandraSqlExecutor
```

Comment:
Could you consider creating a `CassandraCreateAndInsertDataSetup` extends `CreateAndInsertDataSetup`? Then, we can avoid passing column counts and parsing INSERT statement.

Reply:
Good idea. Avoiding specifying column count will be more convenient for maintenance.

Reply:
My bad, let's implement `DataSetup` instead of extending `CreateAndInsertDataSetup`.

Reply:
Done.